src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy

```diff
@@ -0,0 +1,287 @@
+package org.openkilda.functionaltests.spec.links
+
+import static groovyx.gpars.GParsPool.withPool
+import static org.junit.Assume.assumeTrue
+import static org.openkilda.functionaltests.extension.tags.Tag.HARDWARE
+import static org.openkilda.testing.Constants.RULES_INSTALLATION_TIME
+import static org.openkilda.testing.Constants.RULES_DELETION_TIME
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.extension.failfast.Tidy
+import org.openkilda.functionaltests.extension.tags.Tags
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.command.switches.DeleteRulesAction
+import org.openkilda.messaging.command.switches.InstallRulesAction
+import org.openkilda.messaging.info.event.IslChangeType
+import org.openkilda.messaging.info.event.SwitchChangeType
+import org.openkilda.model.IslStatus
+import org.openkilda.model.SwitchFeature
+import org.openkilda.testing.model.topology.TopologyDefinition.Isl
+
+import spock.lang.See
+
+@Tags(HARDWARE)
+// virtual env doesn't support round trip latency
+@See("https://github.com/telstra/open-kilda/tree/develop/docs/design/network-discovery")
+class RoundTripIslSpec extends HealthCheckSpecification {
+
+    @Tidy
+    def "A round trip latency ISL doesn't go down when one switch is DEACTIVATED"() {
+        given: "A switch with/without round trip latency ISLs"
+        def roundTripIsls
+        def nonRoundTripIsls
+        def swToDeactivate = topology.activeSwitches.find { sw ->
+            if (sw.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)) {
+                roundTripIsls = topology.getRelatedIsls(sw).findAll {
+                    it.dstSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
+                }
+                nonRoundTripIsls = topology.getRelatedIsls(sw).findAll {
+                    !it.dstSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
+                }
+                roundTripIsls && nonRoundTripIsls
+            }
+        } ?: assumeTrue("Wasn't able to find a switch with suitable links", false)
+
+        when: "Deactivate the switch"
```

Comment:
I don't think that we should call it 'Deactivate the switch' in these scenarios. Deactivating the switch may be understood as if the switch actually became down. In these scenarios it is very important that we actually _lose connection to switch_, but the switch itself remains operable.